ci-operator/jobs/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master-presubmits.yaml

```diff
@@ -1,78 +1,5 @@
 presubmits:
   openshift/cluster-ingress-operator:
-  - agent: kubernetes
-    always_run: true
-    branches:
-    - ^master$
-    - ^master-
-    cluster: build03
-    context: ci/prow/e2e-aws-gatewayapi
-    decorate: true
-    labels:
-      ci-operator.openshift.io/cloud: aws
-      ci-operator.openshift.io/cloud-cluster-profile: aws-3
-      ci.openshift.io/generator: prowgen
-      pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-cluster-ingress-operator-master-e2e-aws-gatewayapi
-    optional: true
-    rerun_command: /test e2e-aws-gatewayapi
-    spec:
-      containers:
-      - args:
-        - --gcs-upload-secret=/secrets/gcs/service-account.json
-        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
-        - --lease-server-credentials-file=/etc/boskos/credentials
-        - --report-credentials-file=/etc/report/credentials
-        - --secret-dir=/secrets/ci-pull-credentials
-        - --target=e2e-aws-gatewayapi
-        command:
-        - ci-operator
-        image: ci-operator:latest
-        imagePullPolicy: Always
-        name: ""
-        resources:
-          requests:
-            cpu: 10m
-        volumeMounts:
-        - mountPath: /etc/boskos
-          name: boskos
-          readOnly: true
-        - mountPath: /secrets/ci-pull-credentials
-          name: ci-pull-credentials
-          readOnly: true
-        - mountPath: /secrets/gcs
-          name: gcs-credentials
-          readOnly: true
-        - mountPath: /secrets/manifest-tool
-          name: manifest-tool-local-pusher
-          readOnly: true
-        - mountPath: /etc/pull-secret
-          name: pull-secret
-          readOnly: true
-        - mountPath: /etc/report
-          name: result-aggregator
-          readOnly: true
-      serviceAccountName: ci-operator
-      volumes:
-      - name: boskos
-        secret:
-          items:
-          - key: credentials
-            path: credentials
-          secretName: boskos-credentials
-      - name: ci-pull-credentials
-        secret:
-          secretName: ci-pull-credentials
-      - name: manifest-tool-local-pusher
-        secret:
-          secretName: manifest-tool-local-pusher
-      - name: pull-secret
-        secret:
-          secretName: registry-pull-credentials
-      - name: result-aggregator
-        secret:
-          secretName: result-aggregator
-    trigger: (?m)^/test( | .* )e2e-aws-gatewayapi,?($|\s.*)
   - agent: kubernetes
     always_run: false
```

Comment:
Just noted that the`always_run` is not updated by `make jobs`, after updating the `always_run:` to `true` manually then run `make jobs` again and it also doesn't revert the change. Looks something wrong with `make jobs` and we might need to update the field manually for `e2e-aws-gatewayapi-conformance` job.

Reply:
Right, seems like [TP noticed this problem too](https://github.com/openshift/ci-tools/pull/4529). Once the fix for the ci-tools is merged they'll follow up with a [PR which updates all impacted presubmits](https://github.com/openshift/release/pull/64342).

Unfortunately I cannot do much here because the presubmits for this PR will fail if I'll manually change the value to `always_run: true` in the job definition. I'll let TP fix this in https://github.com/openshift/release/pull/64342.